test/Driver/color-diagnostics.swift

```diff
@@ -1,4 +1,9 @@
-// RUN: not %target-swiftc_driver -color-diagnostics -emit-executable -o %t %s 2>&1 | %FileCheck %s
+// RUN: not %target-swiftc_driver -color-diagnostics -emit-executable -o %t %s 2>&1 \
+// RUN:     | %FileCheck -check-prefix=CHECK-CD %s
+// CHECK-CD: [0m1 = 2{{$}}
+
+// RUN: not %target-swiftc_driver -no-color-diagnostics -emit-executable -o %t %s 2>&1 \
```

Comment:
To really be definitive we'd have to run in an environment that did have colors (so that `-no-color-diagnostics` would be a change), but it's not like this is a load-bearing feature.

Reply:
Yes, I understand that, but I indeed don't see a simple way of simulating that kind of environment.